src/base/bittorrent/private/filterparserthread.cpp

```diff
@@ -238,15 +248,19 @@ int FilterParserThread::parseDATFilterFile()
                 ++ruleCount;
             }
             catch (std::exception &e) {
-                LogMsg(tr("IP filter exception thrown for line %1. Exception is: %2").arg(nbLine)
-                                               .arg(QString::fromLocal8Bit(e.what())), Log::CRITICAL);
+                parseErrorCount++;
+                if (parseErrorCount < MAX_LOG_LINES)
+                    LogMsg(tr("IP filter exception thrown for line %1. Exception is: %2").arg(nbLine)
+                           .arg(QString::fromLocal8Bit(e.what())), Log::CRITICAL);
             }
         }
 
         if (start >= dataSize)
             offset = 0;
     }
 
+    if (parseErrorCount >= MAX_LOG_LINES)
+        LogMsg(tr("Extra %1 IP filter parsing errors occurred.", "Extra 513 IP filter parsing errors occurred.").arg(parseErrorCount), Log::CRITICAL);
```

Comment:
Shouldn't it be `parseErrorCount  - MAX_LOG_LINES` to show **extra** error (not showing before)?